src/backend/distributed/commands/utility_hook.c

```diff
@@ -427,42 +426,6 @@ ProcessUtilityInternal(PlannedStmt *pstmt,
 		}
 	}
 
-	/*
-	 * TRANSMIT used to be separate command, but to avoid patching the grammar
-	 * it's now overlaid onto COPY, but with FORMAT = 'transmit' instead of the
-	 * normal FORMAT options.
-	 */
-	if (IsTransmitStmt(parsetree))
```

Comment:
Ah, I had noted this but you realized before I pushed my comment :) Great to see we remove